utils/dask.conf

```diff
@@ -1,11 +1,14 @@
-ENVNAME cudf
+ENVNAME rapids
+
+ARCH GPU
 
 NWORKERS 8
 
-12.34.567.890 MASTER
+12.34.567.891 MASTER
+11.12.131.415 WORKER
```

Comment:
These IP addresses look super realistic 🤣

Reply:
Dude what gave it away?